### DISCUSSION TITLE
junction junkie trait incorrect description

### DISCUSSION THREAD

**Discussion by CodeDivingAnon** [FakeFishGames/Barotrauma#15383] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Junction junkie trait (engineer, 1st tier) has some messed up wording, i.e. "permanently repair electrical devices +[amount]% faster"

### Reproduction steps

1. start game
2. switch to engineer class
3. open traits
4. hover over "junction junkie" trait
5. observe how someone did not proofread

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/0e7dcd5fe4885ae561fd39d288f56282afcaf459 (commit in our private repo), fix coming up in a future update.
